### ISSUE TITLE
Improve management of error handling

### ISSUE THREAD

**Issue by andrewholgate** [wfp/node-html-auditor#23]
At present, when executing any of the commands (eg `a11y-audit`), an error is throw:

```
$ a11y-audit
/home/ubuntu/node-html-auditor/bin/a11y-audit:39
  throw new Error('Mandatory arguments are missing a11y-audit --path [path/to/file(s)] --standard [STANDARD]'.red);
  ^

Error: Mandatory arguments are missing a11y-audit --path [path/to/file(s)] --standard [STANDARD]
```

We should improve this and handle the throw error correctly by only displaying:
```
Mandatory arguments are missing a11y-audit --path [path/to/file(s)] --standard [STANDARD]
```
